server/public/partials/statblock/statBlock.html

```diff
@@ -152,12 +152,12 @@ <h4>Languages</h4>
           <span ng-repeat="language in creature.stats.languages"><span ng-if="$index>0">, </span>{{language}}</span>
         </p>
       </div>
-      <div class="property-line" style="margin-bottom: 1.33em">
-        <div class="property-line" style="float: left; margin-left: -1em">
+      <div class="property-line">
```

Comment:
Switched to left/right blocks. Checked new design on my PC and mobile and it looks good